src/components/inputs/code-input/code-input.scss

```diff
@@ -60,3 +60,33 @@
     }
   }
 }
+
+@media (max-width: #{$breakpoint-base-lg}) {
+  .responsive input {
```

Comment:
You need to change this line and others, removing `input` from inside. 
The comment added was only a suggestion to change next lines.